### PR TITLE
Homepage without sign in form

### PR DESCRIPTION
**Pull request by brunomichetti**
Screen Shot 2020-06-03 at 17 28 07

HomePage without sign-in form

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -1,38 +1,113 @@
-.App {
+body {
+  margin: 0%;
+}
+
+
+.main-div{
```
Comment (manuelcheveste) [inline src/App.css]:
```suggestion
.main-div {
```

**Review by manuelcheveste — APPROVED** [review]

File: src/App.js [hunk 2]
```diff
@@ -1,24 +1,41 @@
...
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+    <div class='main-div'>
+        <div class='left-container'>
+          <div class='menu'>
+            <img src={menu_icon} alt='menu'></img>
```
Comment (Alxzu) [inline src/App.js]: If you aren't planning to put something inside the tag you could just do it one self close tag, like this... 
```suggestion
            <img src={menu_icon} alt='menu' />
```

**Review by Alxzu — APPROVED** [review]
Awesome first PR 👏 
- Read [this guideline](http://getbem.com/) to have a better understanding of how to model your css code
- For sizing fonts and elements it's better using rems or ems instead of pxs [see more here](https://medium.com/code-better/css-units-for-font-size-px-em-rem-79f7e592bb97)
- There are too many "magic numbers" as  `bottom: 57%` or `padding-top: 9.11%`... I'll suggest using whether a grid system, maybe bootstrap would fit for this job, or try not to use those in favor of round them up (let's say creating a class `container` with padding: 16px.... and use that class to wrap  some things

**Comment by brunomichetti** [comment]
> Awesome first PR 👏
> 
> * Read [this guideline](http://getbem.com/) to have a better understanding of how to model your css code
> * For sizing fonts and elements it's better using rems or ems instead of pxs [see more here](https://medium.com/code-better/css-units-for-font-size-px-em-rem-79f7e592bb97)
> * There are too many "magic numbers" as  `bottom: 57%` or `padding-top: 9.11%`... I'll suggest using whether a grid system, maybe bootstrap would fit for this job, or try not to use those in favor of round them up (let's say creating a class `container` with padding: 16px.... and use that class to wrap  some things

Thanks for the tips!
I'll take a look at those points.
The "magic numbers" were calculated in this way:
In Figma, I took the size of the container, and the size of the corresponding item that I wanted to calculate, and then I applied "regla de 3" to find the percentage. But I'll take a look at those points you recommended.

File: src/App.js [hunk 3]
```diff
@@ -1,24 +1,41 @@
 import React from 'react';
-import logo from './logo.svg';
 import './App.css';
+import smilies from './assets/smilies.png';
+import i6 from './assets/i6.png';
+import social from './assets/social.png';
+import play from './assets/play.png';
+import appstorebtn from './assets/appstore_button.png';
+import menu_icon from './assets/icons/menu_icon.png';
```
Comment (tamarasuarezrod) [inline src/App.js]: use camel case pls

File: src/App.css [hunk 4]
```diff
@@ -1,38 +1,117 @@
-.App {
+body {
+  margin: 0%;
```
Comment (tamarasuarezrod) [inline src/App.css]: isn't 0 the default value?

File: src/App.css [hunk 5]
```diff
@@ -1,38 +1,117 @@
...
+}
+
+
+.main-div {
+  display: flex;
+  height: 100vh;
+  border: 0;
+  padding: 0;
```
Comment (tamarasuarezrod) [inline src/App.css]: same here

File: src/App.css [hunk 6]
```diff
@@ -1,38 +1,117 @@
...
+
+
+.left-container {
+  background-color: white;
+  width: 100%;
+}
+
+
+.right-container {
+  background-color: #2FBCF7;
```
Comment (tamarasuarezrod) [inline src/App.css]: get the colors from a constant file

File: src/App.css [hunk 7]
```diff
@@ -1,38 +1,117 @@
...
+  font-family: Open Sans Bold;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 18px;
+  line-height: 32px;
+}
+
+
+.description-text {
+  font-family: Open Sans Regular;
```
Comment (tamarasuarezrod) [inline src/App.css]: is this working?

File: src/App.css (hunk 6 above)
Comment (Alxzu) [reply src/App.css]: I think you can't do that on pure CSS right?

**Review by manema — APPROVED** [review]
Looking good!
I strongly recommend you add some css preprocessor like sass now that you are in an early stage, It will give you many useful tools:
- Variables: Colors, shadows, font sizes.. there are many piece of style you will be able to reuse along your app, all these can be set as constants in a file to be used later on.
- Nest classes: you will need it whether you will follow alexis advice regarding using BEM.
-...

https://sass-lang.com/guide
A little tutorial to add it to create react apps: https://scotch.io/starters/react/adding-sass-to-create-react-app-applications

File: src/App.css (hunk 4 above)
Comment (brunomichetti) [reply src/App.css]: I had to add the that because otherwise a white border appears

File: src/App.css (hunk 7 above)
Comment (brunomichetti) [reply src/App.css]: Yes, I defined a font-family, and used it setting the downloaded font

File: src/App.css (hunk 6 above)
Comment (tamarasuarezrod) [reply src/App.css]: Oh, I didn't know that

**Review by tamarasuarezrod — APPROVED** [review]

**Pull request merged**